pkg/server/smb/server.go

```diff
@@ -101,11 +131,17 @@ func (s *Server) RemoveSmbGlobalMapping(context context.Context, request *intern
 		return response, fmt.Errorf("remote path is empty")
 	}
 
-	err := s.hostAPI.RemoveSmbGlobalMapping(remotePath)
+	mappingPath, err := getRootMappingPath(remotePath)
```

Comment:
does the value of `mappingPath` change after the call on line 66?

Reply:
lines 66 and 134 relate to different methods

Reply:
Got it, thanks